src/main/java/com/sss/garage/model/league/League.java

```diff
@@ -32,6 +32,10 @@ public class League {
     @OneToMany(mappedBy="league")
     private Set<Split> splits;
 
+    private Boolean active = false;
+
+    private String discordGroupId;
```

Comment:
U nas każda liga ma osobną grupe? W sensie liga jesienna i wiosenna to różne grupy czy ta sama?

Reply:
ID grupy to w tym przypadku ID rangi na discordzie. Lepiej będzie zrobić różne, jakby rangi się miały zmienić